libs/cardano-ledger-conformance/test/Test/Cardano/Ledger/Conformance/Imp.hs

```diff
@@ -12,16 +14,27 @@ module Test.Cardano.Ledger.Conformance.Imp (spec) where
 
 import Cardano.Ledger.Alonzo.Tx (AlonzoTx)
 import Cardano.Ledger.BaseTypes
+import Cardano.Ledger.Coin (Coin (..))
 import Cardano.Ledger.Conway (Conway)
+import Cardano.Ledger.Conway.Core (
```

Comment:
This type is aught the be imported from the package that defines it
Core modules are designed to mostly export type classes with type families and lenses
```suggestion
import Cardano.Ledger.Babbage.TxBody (
```